packages/block-library/src/comments-pagination-next/block.json

```
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/comments-pagination-next",
+	"title": "Comments Pagination Next",
+	"category": "design",
```

Let's follow other inner blocks used with the Comment Template block:
```suggestion
	"category": "theme",
```